subscriptions/defaults.py

```diff
@@ -27,3 +27,4 @@
 DEFAULT_SUBSCRIPTIONS_CACHE_NAME = 'subscriptions'
 DEFAULT_SUBSCRIPTIONS_CURRENCY = 'USD'
 DEFAULT_SUBSCRIPTIONS_TRIAL_PERIOD = relativedelta(seconds=0)
+DEFAULT_ABANDON_PENDING_PAYMENTS_AFTER = timedelta(days=3)
```

Comment:
How about 9 days? Or even better – not abandoning them automatically at all, just listing them periodically e.g. once a day. That would require a human interaction and his "fault" at determining whether a pending payment is to be abandoned.

This is payment. It's on the Paddle side to determine whether it's a success or a failure. And their default subscription refresh points to three retries, three days apart each. And there's no actual explanation on how "pending" works, so maybe leaving it as "pending" is the right thing to do.

Reply:
No, there is no "default 3 retries" on paddle side, it doesn't apply to one-off charges:
> There is no built-in dunning for the Charges API, so you will need to manage error handling and retries on your own
https://developer.paddle.com/classic/api-reference/23cf86225523f-create-one-off-charge

Anyway I agree to leave it as PENDING.